fusedrug/data/tokenizer/modulartokenizer/multi_tokenizer.py

```diff
@@ -707,20 +826,38 @@ def encode_list(
 
         merged_encoding = Encoding.merge(encoded_list)
 
+        if max_len is None:
+            if self.max_len is not None:
+                max_len = self.max_len
+
         if max_len is not None:
             merged_encoding.truncate(max_length=max_len)
-        if padding_token_id is not None:
-            # TODO: find the actual padding token ID
-            rnd_type = list(self.tokenizers_info.keys())[0]
-            rnd_inst = self.tokenizers_info[rnd_type]["tokenizer_inst"]
+
+        rnd_type = list(self.tokenizers_info.keys())[0]
```

Comment:
what does rnd_type mean ?

Reply:
Removed those. The point was to access any of the sub-tokenizers and call its respective token_to_id (since padding token is special, it doesn't matter which sub-tokenizer to use). I changed it to call self.token_to_id.